bin/deduplicate.py

```diff
@@ -66,13 +76,19 @@ def main(fastas, outdir):
             " files."
         )
     )
-    parser.add_argument("-f", "--fastas", help="The fasta files paths", nargs="+")
+    input_group = parser.add_mutually_exclusive_group()
+    input_group.add_argument("-f", "--fastas", help="The fasta files paths", nargs="+")
```

Comment:
Could you please indicate in the descriptions that these are mutually exclusive arguments and when to use each?